### DISCUSSION TITLE
Bltouch Probing in mid air does not move down.

### DISCUSSION THREAD

**Discussion by molesza** [Jyers/Marlin#274] [Q&A] [answered]
Hi

I've flashed the 15x15 UBL firmware and Bltouch 5x5 firmware and am seeing strange behavior with my Bltouch.

If I home the printer it homes x and y and goes to the center of the bed. It will then go up on Z as is usual with a bltouch. Then the probe pops out and back in a few times and the screen of the ender 3 v2 shows "STOPPED."

If I try and do a bed level it does the same and then moves to the bottom corner and the probe pops in and out over and over until I switch the printer off.

I made sure all the cables were plugged in correctly to the mainboard and the bltouch.

Hope someone can help!

**Comment by Jyers** [comment]
Is your BL Touch connected to the z endstop pin or the dedicated BL Touch port? The precompiled .bins are configured to use the dedicated port, so if your BL Touch is plugged into the endstop pins it will not function properly.

**Reply by molesza** [reply]
Thanks Jyers for the reply. Its connected to the Bltouch dedicated port. I did not need to unplug my z endstop.

**Reply by Jyers** [reply]
If you go to the move menu, are you able to move your z-axis?

**Reply by molesza** [reply]
I seem to remember being able to do that but I'll confirm in a few hours. Thanks

**Reply by molesza** [reply]
No problem at all using move to move x,y and z. Maybe my Bltouch is faulty because I am not seeing anyone else having the same problem.

**Reply by Jyers** [reply]
I've seen this problem many times, but it was always been due to an incorrectly connected probe. So most likely either the probe is faulty or the connection is.

**Reply by molesza** [reply]
I was able to use the z axis. Everything works fine except the bltouch. I have since flashed the firmware without bltouch and manually leveled. Would love to use the bltouch though. I'm happy to keep troubleshooting but I feel my bltouch may be faulty.

**Reply by Jyers** [reply]
Unfortunately that would be my best guess. Have you tried any other firmwares before mine? As if one of those worked with the BL Touch that could mean it is simply a config issue.

**Comment by Jyers** [comment]
Any update on this?

**Reply by AhJuicyKiwi** [reply]
I've been following as well since I just updated to the 1.2.2 firmware. I noticed I can't move Z at all besides up. So therefore Ill have to plug in the Z Axis Stop? I'ma little confused still any help would be appreciated thank you!

**Comment by sn0opy** [comment, marked as answer]
I had similar issues with the most recent Smith3D firmware as well and thought it was some copy&paste issue or even a Marlin issue. I went back to the January firmware which did work just fine back then. After realizing, that bed leveling even failed for that one, I double checked the BLTouch connector(s) and just re-applied some hot glue. 

After that, I went back to the latest 5x5 HS BLtouch firmware from @Jyers and everything seems to work just fine again.

In other words: double-check your cable connections.
